packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -1,220 +1,203 @@
 <template>
 	<div :class="formGroupStyle">
-		<div :class="decreaseButtonStyle" @click="decrementValue">
+		<div :class="decrementButtonClasses" @click="decrementValue">
 			<i :class="yocoClassDecrement">minus</i>
 		</div>
-		<input
-			type="number"
-			:min="min"
-			:max="max"
-			:step="step"
-			:class="inputStyle"
-			:disabled="disabled"
-			:value="currentValue"
-			@input="handleInput($event.target.value)"
-		/>
-		<div
-			v-if="max && showMaxNumber"
-			class="kt-input-number__max"
-			v-text="max"
-		/>
-		<div :class="increaseButtonStyle" @click="incrementValue">
+		<div :class="middleClasses" @click="$refs.input.focus()">
+			<input
+				ref="input"
+				type="text"
+				:class="inputClasses"
+				:disabled="disabled"
+				:max="max"
+				:min="min"
+				:step="step"
+				:value="currentValue"
+				@input="handleInput($event.target.value)"
+			/>
+			<div v-if="showMaxNumber">/</div>
+			<div v-if="showMaxNumber" class="kt-input-number__max" v-text="max" />
+		</div>
+		<div :class="incrementButtonClasses" @click="incrementValue">
 			<i :class="yocoClassIncrement">plus</i>
 		</div>
 	</div>
 </template>
 
 <script>
+const DECIMAL_PLACES = 3
+const DECIMAL_SEPARATOR = (1.1).toLocaleString().substring(1, 2)
```

Comment:
fancy

Reply:
@carsoli actually, I think we should probably change this to `(1.1).toLocaleString().replace(/\d/g, '')`, or what do you think? Some languages have really odd decimal separators.